cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala

```diff
@@ -19,10 +19,10 @@ case class DrsPathResolver(drsConfig: DrsConfig, httpClientBuilder: HttpClientBu
 
   implicit lazy val urlDecoder: Decoder[Url] = deriveDecoder
   implicit lazy val checksumDecoder: Decoder[ChecksumObject] = deriveDecoder
-  implicit lazy val dataObjectDecoder: Decoder[DosDataObject] = deriveDecoder
-  implicit lazy val dosObjectDecoder: Decoder[DosObject] = deriveDecoder
+  implicit lazy val dataObjectDecoder: Decoder[DrsDataObject] = deriveDecoder
+  implicit lazy val drsObjectDecoder: Decoder[DrsObject] = deriveDecoder
   implicit lazy val saDataObjectDecoder: Decoder[SADataObject] = deriveDecoder
-  implicit lazy val marthaResponseDecoder: Decoder[MarthaResponse] = deriveDecoder
+  implicit lazy val marthaResponseDecoder: Decoder[MarthaResponse] = Decoder.forProduct2("dos", "googleServiceAccount")(MarthaResponse.apply)
```

Comment:
Should this still say `"dos"`? If it's for backward compatibility, a code comment would be worthwhile so a future maintainer doesn't remove it.

Reply:
@salonishah11 can correct me if I'm wrong but I believe Martha is still returning a JSON value keyed by "dos" instead of using the current "drs" terminology. I believe this Circeism says "when decoding objects of this type, take the value keyed by "dos" and pass that as the first argument to `MarthaResponse.apply`"

Reply:
Aha, makes sense to me.

I'm curious why we are renaming everything without also accommodating Martha itself changing from `dos` to `drs`, i.e. look for `drs` and fall back to `dos` if it doesn't exist.

That's just general engineering intuition and I don't know enough about this project to say, "yes, we must do this" - so this remains a TOL.

Reply:
Sounds good as an enhancement request since I assume Martha will one day get with the times and we should be ready for that.